Documentation/ContentBlocks/Carousel.rst

```diff
@@ -0,0 +1,182 @@
+:navigation-title: Carousel
+..  include:: /Includes.rst.txt
+
+..  _carousel:
+
+==============================
+Carousel Content Block example
+==============================
+
+This content block demonstrates some additional features of
+:composer:`friendsoftypo3/content-blocks`. We assume that you are already
+familiar with the concepts described in chapter
+`Custom Content Blocks <https://docs.typo3.org/permalink/t3sitepackage:content-blocks>`_.
+
+..  contents:: Table of contents
+
+..  _carousel-directory:
+
+Directory structure of the carousel content element
+===================================================
+
+Additionally to the files that the jumbotron (compare
+:ref:`content-blocks-jumbotron-directory`) the carousel comes with special CSS
+and JavaScript needed for this element only.
+
+Additionally it supplies a template for its display in the backend.
+
+..  directory-tree::
+    :level: 2
+    :show-file-icons: true
+
+    *   packages/my_site_package/ContentBlocks/ContentElements/jumbotron
+
+        *   assets
+
+            *   frontend.css
+            *   frontend.js
+            *   icon.svg
+
+        *   language
+
+            *   labels.xlf
+
+        *   templates
+
+            *   backend-preview.html
+            *   frontend.html
+
+        *   config.yaml
+
+..  _carousel-configuration:
+
+The carousel: A Content Block containing a collection of elements
+=================================================================
```

Comment:
```suggestion
===============================================================
```

Reply:
The underline is as long as the text already?

Reply:
Seems like GitHub has no true monospace Font, weird.